libs/designsystem/src/lib/helpers/color-helper.ts

```diff
@@ -143,3 +143,9 @@ export interface KirbyColor extends Color {
   shade: Color;
   contrast: Color;
 }
+
+export enum NotificationColor {
+  success = 'success',
+  warning = 'warning',
+  danger = 'danger',
+}
```

Comment:
:fire: Don't repeat yourself - this already exists in `ColorHelper` via `color-helper.styles` auto-generated from scss
```suggestion
```

Reply:
No an enum like this doesn't exist to use for type safety doesn't exist.

Reply:
Then add it to the autogeneration of  `color-helper.styles`

Reply:
I don't think we can generate an enum from the sass to ts tool.

Reply:
String literal is an option